src/client/features/view/components/graph/index.js

```diff
@@ -62,23 +63,20 @@ class Graph extends React.Component {
   render() {
     if (!this.state.graphEmpty) {
       return (
-        <div className='Graph'>
-          <div
-            id={this.state.graphId}
-            ref={dom => this.graphDOM = dom}
-            style={{
-              width: this.state.width,
-              height: this.state.height
-            }}
-          />
-        </div>
+        h('div.Graph', [
+          h('div', {
+            'id': this.state.graphId,
```

Comment:
@hmtinc @jonahdlin @gaelder 

Pretty sure you don't need to have object keys as strings.  Maybe if the key shares a keyword like class, but in general you shouldn't have to write them as strings.